web/src/message_list_tooltips.ts

```diff
@@ -193,7 +204,7 @@ export function initialize() {
             // Handle dynamic "starred messages" and "edit" widgets.
             const $elem = $(instance.reference);
             const tippy_content = $elem.attr("data-tippy-content");
-            const $template = $(`#${CSS.escape($elem.attr("data-tooltip-template-id"))}`);
+            const $template = $(`#${CSS.escape($elem.attr("data-tooltip-template-id")!)}`);
             instance.setContent(tippy_content ?? parse_html($template.html()));
```

Comment:
I think the `!` might not be right here; I think we only expect `data-tooltip-template-id` to be present if `tippy_content` is not valid. Might need a prep commit to rewrite this logic to not generate an invalid `$template` in the event that `data-tippy-content` was present, using if/else clauses or something.